### ISSUE TITLE
Epoch times on image and Json dont match

### ISSUE THREAD

**Issue by EccentricDyslexic** [openalpr/openalpr#102]
Forgive me if i have overlooked something obvious, as in the title, for example, this is the epoch time at the end of the picture file name from just now - 1429079533274
and this is the epoch returned by Json - 1429079533099 for the same plate.

Cheers

Steve

**Comment by matthill** [comment]
Can you upload the full JSON output?

**Comment by EccentricDyslexic** [comment]
{u'uuid': u'frostfields-cam1-1429386755851', u'data_type': u'alpr_results', u'epoch_time': 1429386755477, u'site_id': u'frostfields', u'img_height': 576, u'img_width': 704, u'results': [{u'plate': u'FD0GVCX', u'confidence': 88.063858, u'region_confidence': 0, u'region': u'', u'plate_index': 0, u'processing_time_ms': 122.898407, u'candidates': [{u'matches_template': 0, u'plate': u'FD0GVCX', u'confidence': 88.063858}], u'coordinates': [{u'y': 66, u'x': 426}, {u'y': 55, u'x': 562}, {u'y': 95, u'x': 565}, {u'y': 107, u'x': 428}], u'matches_template': 0, u'requested_topn': 0}], u'version': 2, u'camera_id': 1, u'processing_time_ms': 373.898773, u'regions_of_interest': [{u'y': 0, u'x': 0, u'height': 576, u'width': 704}]}

**Comment by EccentricDyslexic** [comment]
Hi Matt, did the image come through with the Json ok?
Cheers
Steve

**Comment by matthill** [comment]
Steve,

I see the issue where the epoch time (1429386755477) is different than the uuid (frostfields-cam1-1429386755851).  The file on disk does match the UUID.  Is that also what you're seeing?

**Comment by EccentricDyslexic** [comment]
Hi Matt, 

What do you mean by the file on disk Matt?  The saved image name is frostfields-cam1-1429386755034.jpg is the name the uuid?  am i missing something here lol

Steve

**Comment by matthill** [comment]
Yes, there should be a file on disk in the storage location referenced in alprd.conf named frostfields-cam1-1429386755851.jpg (matching the UUID).  Is there not?

**Comment by EccentricDyslexic** [comment]
No, the matching file is named frostfields-cam1-1429386755034.jpg quite odd.

Steve

**Comment by matthill** [comment]
Hmm, I can't seem to reproduce that problem.  Is this for every single result, or just some of them?

This is 64-bit Linux?  Compiled yourself or using the pre-compiled binaries?

**Comment by EccentricDyslexic** [comment]
Every result.  I used the easy way to install, and yes 64 Ubuntu.  I am getting json from beanstalk if that makes any difference.

Cheers

Steve
